spec/bambora/client_spec.rb

```diff
@@ -4,8 +4,21 @@
 
 module Bambora
   describe Client do
-    describe '#profiles' do
-      pending
+    describe '#profile' do
+      context 'with defaults' do
+        subject { Bambora::Client.new }
+
+        it 'returns a Bambora::V1::Profile' do
+          expect(subject.profile).to be_a Bambora::V1::Profile
+        end
+      end
+
+      context 'with a different version' do
+        subject { Bambora::Client.new(version: 'V2') }
+
+        it 'returns a raises an error' do
```

Comment:
is something missing here(body/pending + rewording)?

Reply:
Oof, sorry! Just pushed this!